server/services/discord/actions/on_join_voice_channel.js

```diff
@@ -8,16 +9,22 @@ const client = require('../init').client
  * @param {*} newChannel Current channel object
  */
 client.on('voiceStateUpdate', (oldChannel, newChannel) => {
-  var oldUserChannel = oldChannel.channelID
-  var newUserChannel = newChannel.channelID
-  if (!newUserChannel) {
-    //  Leaving a voice channel
-    console.log('User left the channel')
-    return
-  }
+  try {
+    console.log('A new user just joined the Discord server: ' + user.tag)
+    var oldUserChannel = oldChannel.channelID
+    var newUserChannel = newChannel.channelID
+    if (!newUserChannel) {
+      //  Leaving a voice channel
+      console.log('User left the channel')
```

Comment:
Need to replace it with the Glue no ?

Reply:
We do not have an action based on a user leaving a channel.